src/components/Button/index.jsx

```diff
@@ -83,6 +84,7 @@ const Button = forwardRef(
           'transition--default',
           `fdsButton--${size}`,
           `fdsButton--${theme}`,
+          ...(margin ? [`margin--${margin}`] : []),
```

Comment:
[could] move this to line 76 in the object: 
 ```
breakoutLink: isBreakoutLink,
`margin--${margin}`: margin
```

Reply:
ah yeah good point! Done.